man/mysqlbinlog.1

```diff
@@ -1731,8 +1724,7 @@ LOG_EVENT_BINLOG_IN_USE_F
 T}:T{
 Log file correctly closed\&. (Used only in
 FORMAT_DESCRIPTION_EVENT\&.) If
-this flag is set (if the flags are, for example,
-\'01 00\') in a
+this flag is set (if the flags are, for example, '01 00') in a
```

Comment:
This one changes from
```
           this flag is set (if the flags are, for example,
           '01 00') in a
           this flag is set (if the flags are, for example, '01 00') in a
```
to
```
this flag is set (if the flags are, for example, '01 00') in a
```
Which probably is better?

Reply:
There was line changes to before using backslash to escape singe quotes. But that was completely unnecessary so I cleaned it away and re-flowed the paragraph with normal line wrapping at 80 chars.

Reply:
This renders correctly now.
![image](https://github.com/MariaDB/server/assets/668724/717d7be2-c037-4cd0-addf-a3f39db82236)

Reply:
Meld of ASCII rendering of this section before and after shows fixing man page syntax error did not affect the text other than having nor more readable line wrapping in ASCII (but actual man page re-flowed automatically both before and after).

![image](https://github.com/MariaDB/server/assets/668724/63743307-f939-49fd-a2fe-d0a8bdc42717)